lib/components.rb

```diff
@@ -54,7 +54,8 @@ def list_components(repos)
   status_heading = "Status" + "".ljust(15)
   out = "#{name_heading} #{status_heading} Created\n".bold
   repos.each do |repo_name, config|
-    process = running_info[repo_name] || {"state" => "missing"}
+    process = running_info[repo_name] 
+    process = container_info[repo_name] || {"state" => "missing"} if process.nil?
     running = case process["state"]
```

Comment:
If (and only if) I understand how this works, then the first assignment (`process = running_info[repo_name]` is always overwritten by the ternary `process = container_info[repo_name] || {"state" => "missing"} if process.nil?` that immediately follows.

Have I understood the ternary correctly?

Reply:
Oooh! I didn't understand. The `||` which I read as shorthand for a ternary 'or' is actually a merge of two objects?

Reply:
Ok this probably makes sense.